orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml

```diff
@@ -23,9 +23,15 @@
   #  when: preburn
 
 - name: Add JFrog key
-  apt_key:
-    data: "{{ lookup('file', 'jfrog.pub') }}"
-    state: present
+  block:
+    - get_url:
```

Comment:
Pretty sure you didn't have to store it in a variable and you could have just used `file:` in the apt_key command